pkg/kubelet/util.go

```diff
@@ -18,11 +18,20 @@ package kubelet
 
 import (
 	"fmt"
+	"sort"
 
+	"github.com/golang/glog"
 	"k8s.io/kubernetes/pkg/api"
 	"k8s.io/kubernetes/pkg/capabilities"
+	kubecontainer "k8s.io/kubernetes/pkg/kubelet/container"
 	kubetypes "k8s.io/kubernetes/pkg/kubelet/types"
 	"k8s.io/kubernetes/pkg/securitycontext"
+	"k8s.io/kubernetes/pkg/util/wait"
+)
+
+const (
+	// The limit on the number of buffered container deletion requests
+	containerDeletionWorkerBufferLimit = 10
```

Comment:
I copied from podKillingChannelCapacity. I've added a comment to indicate the number is arbitrary.